proto/vmexec.proto

```diff
@@ -62,6 +69,50 @@ message ExecResponse {
   google.rpc.Status status = 4;
 }
 
+message ExecStreamedRequest {
+  oneof Message {
```

Comment:
@tylerwilliams I thought you didn't like oneofs?
I don't have a strong feeling either way, but we should have a consistent approach on whether oneofs should or should not be used.

Reply:
I am happy to not use a `oneof`, since the Go API is gnarly. Looks like we don't use `oneof` for `RegisterAndStreamWorkRequest` so I'll update this to match.

Reply:
+1 to not using oneof, I missed this